Tests/GRPCTests/GRPCReflectionServiceTests/ReflectionServiceUnitTests.swift

```diff
@@ -117,134 +117,199 @@ final class ReflectionServiceUnitTests: GRPCTestCase {
   func testNameOfFileContainingSymbolEnum() throws {
```

Comment:
It would be great to add some integration tests which validate we get appropriate error responses too. Feel free to do it in a separate PR if you prefer.